ftw/linkchecker/plone_site.py

```diff
@@ -0,0 +1,47 @@
+from AccessControl.SecurityManagement import newSecurityManager
+from Acquisition import aq_parent
+from ftw.linkchecker import LOGGER_NAME
+from zope.component.hooks import setSite
+from zope.globalrequest import setRequest
+import AccessControl
+import logging
+
+
+class PloneSite():
```

Comment:
The name is a little confusing. But I can't think of a better one right now.

Reply:
You could use `PerSiteConfiguration` ... but I can't see where this class is used actually?!

Reply:
@djowett-ftw 
https://github.com/4teamwork/ftw.linkchecker/blob/4d7cf9447303657193717774cab7645fd50b6473/ftw/linkchecker/command/checking_links.py#L34-L40

Reply:
I refactored the class to PerSiteConfiguration in b6ede20. See @Nachtalb 's comment for the question about the usage.